### PR TITLE
build: improve Prisma setup

### PR DESCRIPTION
**Pull request by mschilling**
* migrate prisma schema (+ related stuff) to project folder
* now ready to configure multiple prisma schemas

### CODE REVIEW

File: libs/foosball/data/.eslintrc.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["../../../.eslintrc.json"],
-  "ignorePatterns": ["!**/*"],
+  "ignorePatterns": ["!**/*","generated/**"],
```
Comment (mschilling) [inline libs/foosball/data/.eslintrc.json:3]: excluding generated files from linter

File: package.json [hunk 2]
```diff
@@ -6,7 +6,10 @@
     "nx": "nx",
     "start": "nx serve",
     "build": "nx build",
-    "test": "nx test"
+    "test": "nx test",
+    "postinstall": "npm run generate-prisma-clients",
```
Comment (mschilling) [inline package.json:10]: For now: this is the suggested way to do the job

File: package.json [hunk 3]
```diff
@@ -17,7 +20,8 @@
     "@nestjs/jwt": "^8.0.0",
     "@nestjs/passport": "^8.0.1",
     "@nestjs/platform-express": "^8.0.0",
-    "@prisma/client": "^3.11.1",
+    "@nx-tools/nx-prisma": "^2.0.0-alpha.4",
```
Comment (mschilling) [inline package.json:23]: dev installs needed. Official version will be released next week :-)

**Review by sanderelderman — APPROVED** [review]

File: apps/foosball/api/project.json [hunk 4]
```diff
@@ -45,7 +45,13 @@
       "options": {
         "jestConfig": "apps/foosball/api/jest.config.js",
         "passWithNoTests": true
-      }
+      },
+      "dependsOn": [
```
Comment (mschilling) [inline apps/foosball/api/project.json:49]: This will make sure that `foosball-data:generate` will be run

File: libs/foosball/.gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+dev.db*
+generated/
```
Comment (mschilling) [inline libs/foosball/.gitignore:2]: For now it seems the way to go to exclude `generated/`. To be evaluated.

**Comment by mschilling** [comment]
🚀

**Pull request merged**